google/ggrc-core
# [5OBJ] GGRC-8529 Extend 5OBJ with Q-side fields

# Dependencies

_**This PR should be merged into [GGRC-7200](https://github.com/google/ggrc-core/tree/GGRC-7200) branch which is the common one for all 5OBJ. PRs**_

This PR is `on hold` until the dependencies are merged:

- [x] #10615 (Move Contracts & Policies out into separate tables)

# Issue description

All 5OBJ should be extended with the following Q-side fields:
- `external_slug` (string, nullable, unique);
- `external_id` (integer, nullable, unique);
- `created_by_id` (integer, nullable).

# Steps to test the changes

1. Run the migration and make sure that all the 5OBJ have Q-side fields with appropriate constraints.

# Solution description

* Add the migration which extends 5OBJ tables with the Q-side fields.

# Sanity checklist


# Migration checklist

# PR Review checklist
